Formula/sonarqube.rb

```diff
@@ -6,7 +6,7 @@ class Sonarqube < Formula
 
   bottle :unneeded
 
-  depends_on :java => "1.8+"
+  depends_on :java => "11+"
```

Comment:
Is that `11` or `1.11`?

Reply:
I see what you mean, and it's `11`. At some point after Java 8 they changed their version numbering and dropped the `1.x`.

```
~ » jenv shell openjdk64-1.8.0.212
------------------------------------------------------------
~ » java -version
openjdk version "1.8.0_212"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_212-b03)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.212-b03, mixed mode)
------------------------------------------------------------
~ » jenv shell openjdk64-11.0.4
------------------------------------------------------------
~ » java -version
openjdk version "11.0.4" 2019-07-16
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.4+11)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.4+11, mixed mode)
------------------------------------------------------------
```

And I get the green checkmark for the `java >= 11` dependency from `brew info`

```
~ » brew info sonarqube
sonarqube: stable 7.9.1
Manage code quality
https://www.sonarqube.org/
Conflicts with:
  sonarqube-lts (because both install the same binaries)
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/sonarqube.rb
==> Requirements
Required: java >= 11 ✔
...
```

Reply:
Thanks, you rock!